packages/lib/modules/pool/PoolDetail/PoolActivityTable/PoolActivityTableRow.tsx

```diff
@@ -58,7 +58,7 @@ function EnsOrAddress({ userAddress, chain }: { userAddress: `0x${string}`; chai
           backgroundColor="background.level4"
           borderRadius="100%"
           height="24px"
-          src={ensAvatar || fallbackSVG.toDataUriSync()}
+          src={ensAvatar || fallbackSVG.toDataUri()}
```

Comment:
Can't do this because `toDataUri` returns a promise. That's why Sync was required. Code will have to be refactored if for some reason we've lost `toDataUriSync`.